### PR TITLE
update synopsis for admin statements

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Provide only the specific synopsis for specific Admin statements 

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb/blob/master/pkg/parser/parser.y

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-admin-show-telemetry.md [hunk 1]
```diff
@@ -15,8 +15,7 @@ The `ADMIN SHOW TELEMETRY` statement shows the information that will be reported
 
 ```ebnf+diagram
 AdminStmt ::=
-    'ADMIN' ( 'SHOW' ( 'DDL' ( 'JOBS' Int64Num? WhereClauseOptional | 'JOB' 'QUERIES' NumList )? | TableName 'NEXT_ROW_ID' | 'SLOW' AdminShowSlow | 'TELEMETRY' ) | 'CHECK' ( 'TABLE' TableNameList | 'INDEX' TableName Identifier ( HandleRange ( ',' HandleRange )* )? ) | 'RECOVER' 'INDEX' TableName Identifier | 'CLEANUP' ( 'INDEX' TableName Identifier | 'TABLE' 'LOCK' TableNameList ) | 'CHECKSUM' 'TABLE' TableNameList | 'CANCEL' 'DDL' 'JOBS' NumList | 'RELOAD' ( 'EXPR_PUSHDOWN_BLACKLIST' | 'OPT_RULE_BLACKLIST' | 'BINDINGS' ) | 'PLUGINS' ( 'ENABLE' | 'DISABLE' ) PluginNameList | 'REPAIR' 'TABLE' TableName CreateTableStmt | ( 'FLUSH' | 'CAPTURE' | 'EVOLVE' ) 'BINDINGS' )
-
+    'ADMIN' 'SHOW' 'TELEMETRY'
```
Comment (qiancai) [inline sql-statements/sql-statement-admin-show-telemetry.md:18]: It seems that this syntax does not exist any more.

File: sql-statements/sql-statement-admin-cancel-ddl.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ The `ADMIN CANCEL DDL` statement also allows you to cancel a DDL job that is com
 
 ```ebnf+diagram
 AdminStmt ::=
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-admin-cancel-ddl.md:16]:
```suggestion
AdminCancelddlStmt ::=
```

File: sql-statements/sql-statement-admin-check-table-index.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ If you use the `ADMIN CHECK INDEX` statement, it only checks the specified index
 
 ```ebnf+diagram
 AdminStmt ::=
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-admin-check-table-index.md:31]:
```suggestion
AdminChecktableindexStmt ::=
```

File: sql-statements/sql-statement-admin-pause-ddl.md [hunk 4]
```diff
@@ -13,7 +13,7 @@ You can use this statement to pause a DDL job that is issued but not yet complet
 
 ```ebnf+diagram
 AdminStmt ::=
-    'ADMIN' ( 'SHOW' ( 'DDL' ( 'JOBS' Int64Num? WhereClauseOptional | 'JOB' 'QUERIES' NumList )? | TableName 'NEXT_ROW_ID' | 'SLOW' AdminShowSlow ) | 'CHECK' ( 'TABLE' TableNameList | 'INDEX' TableName Identifier ( HandleRange ( ',' HandleRange )* )? ) | 'RECOVER' 'INDEX' TableName Identifier | 'CLEANUP' ( 'INDEX' TableName Identifier | 'TABLE' 'LOCK' TableNameList ) | 'CHECKSUM' 'TABLE' TableNameList | 'CANCEL' 'DDL' 'JOBS' NumList | 'PAUSE' 'DDL' 'JOBS' NumList | 'RESUME' 'DDL' 'JOBS' NumList | 'RELOAD' ( 'EXPR_PUSHDOWN_BLACKLIST' | 'OPT_RULE_BLACKLIST' | 'BINDINGS' ) | 'PLUGINS' ( 'ENABLE' | 'DISABLE' ) PluginNameList | 'REPAIR' 'TABLE' TableName CreateTableStmt | ( 'FLUSH' | 'CAPTURE' | 'EVOLVE' ) 'BINDINGS' )
+    'PAUSE' 'DDL' 'JOBS' NumList
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-admin-pause-ddl.md:16]:
```suggestion
    'ADMIN' 'PAUSE' 'DDL' 'JOBS' NumList
```

File: sql-statements/sql-statement-admin-recover.md [hunk 5]
```diff
@@ -10,7 +10,7 @@ When the row data and index data are inconsistent, you can use the `ADMIN RECOVE
 ## Synopsis
 
 ```ebnf+diagram
-AdminCleanupStmt ::=
+AdminStmt ::=
     'ADMIN' 'RECOVER' 'INDEX' TableName IndexName
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-admin-recover.md:14]: https://docs.pingcap.com/tidb/dev/sql-statement-admin#synopsis

File: sql-statements/sql-statement-admin-show-telemetry.md (hunk 1 above)
Comment (qiancai) [reply sql-statements/sql-statement-admin-show-telemetry.md]: According to @bb7133, this syntax has been removed and he will create a PR to remove this doc.

**Review by kennytm — APPROVED** [review]

File: sql-statements/sql-statement-admin-recover.md (hunk 5 above)
Comment (Oreoxmt) [inline sql-statements/sql-statement-admin-recover.md:14]:
```suggestion
    'ADMIN' 'RECOVER' 'INDEX' TableName Identifier
```

File: sql-statements/sql-statement-admin-cancel-ddl.md (hunk 2 above)
Comment (dveeden) [inline sql-statements/sql-statement-admin-cancel-ddl.md:16]:
```suggestion
AdminCancelStmt ::=
```

File: sql-statements/sql-statement-admin-check-table-index.md (hunk 3 above)
Comment (dveeden) [inline sql-statements/sql-statement-admin-check-table-index.md:31]:
```suggestion
AdminCheckStmt ::=
```

File: sql-statements/sql-statement-admin-checksum-table.md [hunk 6]
```diff
@@ -12,7 +12,7 @@ The `ADMIN CHECKSUM TABLE` statement calculates a CRC64 checksum for the data an
 
 ```ebnf+diagram
 AdminStmt ::=
```
Comment (dveeden) [inline sql-statements/sql-statement-admin-checksum-table.md:14]:
```suggestion
AdminChecksumTableStmt ::=
```

File: sql-statements/sql-statement-admin-cleanup.md [hunk 7]
```diff
@@ -10,7 +10,7 @@ The `ADMIN CLEANUP INDEX` statement is used to delete redundant indexes from a t
 ## Synopsis
 
 ```ebnf+diagram
-AdminCleanupStmt ::=
+AdminStmt ::=
```
Comment (dveeden) [inline sql-statements/sql-statement-admin-cleanup.md:13]: I think it should only be called `AdminStmt` if it describes the full `ADMIN...` statement

File: sql-statements/sql-statement-admin-pause-ddl.md [hunk 8]
```diff
@@ -13,7 +13,7 @@ You can use this statement to pause a DDL job that is issued but not yet complet
 
 ```ebnf+diagram
 AdminStmt ::=
```
Comment (dveeden) [inline sql-statements/sql-statement-admin-pause-ddl.md:15]:
```suggestion
AdminPauseStmt ::=
```

File: sql-statements/sql-statement-admin-resume-ddl.md [hunk 9]
```diff
@@ -13,7 +13,7 @@ You can use this statement to resume a paused DDL job. After the resume is compl
 
 ```ebnf+diagram
 AdminStmt ::=
```
Comment (dveeden) [inline sql-statements/sql-statement-admin-resume-ddl.md:15]:
```suggestion
AdminResumeStmt ::=
```

File: sql-statements/sql-statement-admin-check-table-index.md [hunk 10]
```diff
@@ -29,7 +29,7 @@ If you use the `ADMIN CHECK INDEX` statement, it only checks the specified index
...
-    'ADMIN' ( 'SHOW' ( 'DDL' ( 'JOBS' Int64Num? WhereClauseOptional | 'JOB' 'QUERIES' NumList )? | TableName 'NEXT_ROW_ID' | 'SLOW' AdminShowSlow ) | 'CHECK' ( 'TABLE' TableNameList | 'INDEX' TableName Identifier ( HandleRange ( ',' HandleRange )* )? ) | 'RECOVER' 'INDEX' TableName Identifier | 'CLEANUP' ( 'INDEX' TableName Identifier | 'TABLE' 'LOCK' TableNameList ) | 'CHECKSUM' 'TABLE' TableNameList | 'CANCEL' 'DDL' 'JOBS' NumList | 'RELOAD' ( 'EXPR_PUSHDOWN_BLACKLIST' | 'OPT_RULE_BLACKLIST' | 'BINDINGS' ) | 'PLUGINS' ( 'ENABLE' | 'DISABLE' ) PluginNameList | 'REPAIR' 'TABLE' TableName CreateTableStmt | ( 'FLUSH' | 'CAPTURE' | 'EVOLVE' ) 'BINDINGS' )
+    'ADMIN' 'CHECK' ( 'TABLE' TableNameList | 'INDEX' TableName Identifier ( HandleRange ( ',' HandleRange )* )? )
```
Comment (dveeden) [inline sql-statements/sql-statement-admin-check-table-index.md:32]:
```suggestion
    'ADMIN' 'CHECK' ( 'TABLE' TableNameList | 'INDEX' TableName IndexName ( HandleRange ( ',' HandleRange )* )? )
```

File: sql-statements/sql-statement-admin-recover.md (hunk 5 above)
Comment (dveeden) [reply sql-statements/sql-statement-admin-recover.md:14]: If we use Identifier, shouldn't we also do that for the TableName?
```
    'ADMIN' 'RECOVER' 'INDEX' Identifier Identifier
```

We could define `IndexName` and we could define `Index` to consist of a table name and index name.
```
AdminStmt ::=
    'ADMIN' 'RECOVER' 'INDEX' Index

Index ::=
    TableName IndexName

TableName ::=
    Identifier

IndexName ::
    Identifier
```

**Review by dveeden — APPROVED** [review]

**Comment by dveeden** [comment]
image

Should this include v7.5 or not?

File: sql-statements/sql-statement-admin-show-telemetry.md (hunk 1 above)
Comment (dveeden) [reply sql-statements/sql-statement-admin-show-telemetry.md]: https://github.com/pingcap/tidb/pull/51202

**Comment by dveeden** [comment]
This has some overlap with https://github.com/pingcap/docs/pull/17617

**Comment by qiancai** [comment]
> This has some overlap with #17617

yes, you are right. Sorry, I forgot #17617 had fixed the issue. Let me close this PR.

**Pull request closed without merging**